src/Groups/GAPGroups.jl

```diff
@@ -816,16 +1112,20 @@ Return whether `G` is a simple group, i.e.,
 """ issimple
 
 @gapattribute isalmostsimple(G::GAPGroup) = GAP.Globals.IsAlmostSimpleGroup(G.X)
-"""
+@doc Markdown.doc"""
     isalmostsimple(G)
 
-Return whether `G` is an almost simple group, i.e., if `S` < `G` < `Aut(S)` for some non-abelian simple group `S`.
+Return whether `G` is an almost simple group,
+i. e., `G` is isomorphic to a group $H$ with the property
+$S \leq H \leq Aut(S)$, for some non-abelian simple group $S$.
 """ isalmostsimple
 
 """
     ispgroup(G)
 
-Return (``true``,nothing) if `G` is the trivial group, (``true``,``p``) if |`G`| is a non-trivial ``p``-power, (``false``,nothing) otherwise.
+Return `(true, nothing)` if `G` is the trivial group,
+`(true, p)` if |`G`| is a non-trivial power of a prime `p`,
+and `(false, nothing)` otherwise.
```

Comment:
This is potentially inaccurate, as in current GAP versions, `IsPGroup` may also return `true` for infinite p-groups (not that any are available in GAP, as far as I know...)